recipe/meta.yaml

```diff
@@ -49,3 +46,7 @@ about:
 extra:
   recipe-maintainers:
     - conda-forge/event-model
+    - hhslepicka
```

Comment:
And @danielballan?
```suggestion
    - hhslepicka
    - danielballan
```

Reply:
Heh. If @klauer accepts then I guess I do. Though it looks like event-model is working now because I already see a merge button on this repo.